tools/test/sofa_test_scene/hexaBeam.pyscn

```diff
@@ -2,28 +2,62 @@
 import os
 import Sofa
 
-meshPath = os.path.dirname(os.path.abspath(__file__))+'/mesh/'
+plugins=["SofaPython3","SoftRobots","ModelOrderReduction","STLIB",
+         # normally plugin Sofa.Component is enough but still warning
+         "Sofa.Component.Visual",
+         "Sofa.Component.AnimationLoop",
+         "Sofa.GL.Component.Rendering3D",
+         "Sofa.Component.Constraint.Lagrangian.Solver",
+         'Sofa.Component.IO.Mesh',
+         'Sofa.Component.Playback',
+         'Sofa.Component.Constraint.Lagrangian.Correction', # Needed to use components [GenericConstraintCorrection]
+         
+         'Sofa.Component.Engine.Select', # Needed to use components [BoxROI]
+         'Sofa.Component.LinearSolver.Direct', # Needed to use components [SparseLDLSolver]
+         'Sofa.Component.Mapping.Linear', # Needed to use components [BarycentricMapping]
+         'Sofa.Component.Mass', # Needed to use components [UniformMass]
+         'Sofa.Component.ODESolver.Backward', # Needed to use components [EulerImplicitSolver]
+         'Sofa.Component.SolidMechanics.FEM.Elastic', # Needed to use components [TetrahedronFEMForceField]
+         'Sofa.Component.SolidMechanics.Spring', # Needed to use components [RestShapeSpringsForceField]
+         'Sofa.Component.StateContainer', # Needed to use components [MechanicalObject]
+         'Sofa.Component.Topology.Container.Dynamic'] # Needed to use components [TetrahedronSetTopologyContainer]
+
 
 def createScene(rootNode):
 
     rootNode.findData('dt').value=0.02
     rootNode.findData('gravity').value=[0, -9810, 0]
     rootNode.addObject('VisualStyle', displayFlags='showBehaviorModels showForceFields')
-    rootNode.addObject('RequiredPlugin', pluginName='ModelOrderReduction')
-    rootNode.addObject('RequiredPlugin', pluginName='SoftRobots')
+    rootNode.addObject('RequiredPlugin', pluginName="ModelOrderReduction SoftRobots")
+   # rootNode.addObject('RequiredPlugin', pluginName='SoftRobots')
```

Comment:
Maybe you can remove commented code if it's not useful anymore.
Also use the variable plugins to be more concise doing the following:
`rootNode.addObject('RequiredPlugin', pluginName=plugins)`

Reply:
Ah yes, I could remove line 32.
Yes, very good point, it is better to the variable "plugins", ok I will change it.